presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConfig.java

```diff
@@ -43,7 +43,7 @@ public class IcebergConfig
     private List<String> hadoopConfigResources = ImmutableList.of();
     private double minimumAssignedSplitWeight = 0.05;
     private boolean parquetDereferencePushdownEnabled = true;
-    private boolean mergeOnReadModeEnabled;
+    private boolean mergeOnReadModeEnabled = true;
```

Comment:
Please update IcebergDistributedSmokeTestBase::testMergeOnReadDisabled to set the session property.

Reply:
@s-akhtar-baig Fixed.

Reply:
Perfect, thank you!